### ISSUE TITLE
cURL extension for URLRequest

### ISSUE THREAD

**Issue by DimaZava** [SwifterSwift/SwifterSwift#776]
It can be useful extension for ones who want to debug their requests in terminal, Postman, Paw, etc.

```
extension URLRequest {

     /// Returns a cURL command representation of this URL request.
    var curlString: String {

        guard let url = url else { return "" }

        var baseCommand = "curl \(url.absoluteString)"
        if httpMethod == "HEAD" {
            baseCommand += " --head"
        }

        var command = [baseCommand]
        if let method = httpMethod, method != "GET" && method != "HEAD" {
            command.append("-X \(method)")
        }

        if let headers = allHTTPHeaderFields {
            for (key, value) in headers where key != "Cookie" {
                command.append("-H '\(key): \(value)'")
            }
        }

        if let data = httpBody,
            let body = String(data: data, encoding: .utf8) {
            command.append("-d '\(body)'")
        }

        return command.joined(separator: " \\\n\t")
    }
}
```

**Comment by omaralbeik** [comment]
Nice extension! @DimaZava would you please open a PR and other members will review and shape this extension in review

**Comment by LucianoPAlmeida** [comment]
Fixed by #790
